src/olympia/versions/tasks.py

```diff
@@ -0,0 +1,67 @@
+import os
+from base64 import b64encode
+
+from django.core.files.storage import default_storage as storage
+from django.template import loader
+
+from wand.image import Image as WandImage
+
+import olympia.core.logger
+from olympia import amo
+from olympia.amo.celery import task
+from olympia.amo.decorators import write
+
+
+log = olympia.core.logger.getLogger('z.files.utils')
+
+
+def write_svg_to_png(svg_content, out):
+    temp_svg_path = out + '.svg'
+    size = None
+    try:
+        with storage.open(temp_svg_path, 'wb') as svgout:
+            svgout.write(svg_content)
+        with storage.open(temp_svg_path, 'rb') as svgout:
+            with WandImage(file=svgout, format='svg') as img:
+                img.format = 'png'
+                with storage.open(out, 'wb') as out:
+                    img.save(file=out)
+                    size = img.size
+    except IOError as ioerror:
```

Comment:
Any other errors that we should catch? What happens if the svg is broken somehow (we control it, but since it depends on user input...), what kind of exceptions could Wand throw?

Reply:
Looks like they all inherit from http://docs.wand-py.org/en/0.4.4/wand/exceptions.html?highlight=exception#wand.exceptions.WandException so I'll catch that too.

Reply:
https://github.com/mozilla/addons-server/pull/7475/commits/86844e3fbdf9614a69824787c6e4d3ca64a7bf8a